src/api/tests/test_protocol_validations.py

```diff
@@ -1,242 +1,125 @@
-import copy
-
-from django.test import TestCase
-
-from api.models.mermaid import CollectRecord
 from api.submission.protocol_validations import (
     BenthicLITProtocolValidation,
     BenthicPITProtocolValidation,
     FishBeltProtocolValidation,
     HabitatComplexityProtocolValidation,
 )
 from api.submission.validations import ERROR, OK, WARN
-from .data import MockRequest, TestDataMixin
-
-
-class BenthicPITProtocolValidationTest(TestCase, TestDataMixin):
-    def setUp(self):
-        self.load_projects()
-        self.load_project_profiles()
-        self.load_sample_events()
-        self.load_benthicattributes()
-
-        observations = [
-            dict(attribute=str(self.benthic_attribute1a.id), interval=5),
-            dict(attribute=str(self.benthic_attribute1a.id), interval=10),
-            dict(attribute=str(self.benthic_attribute1a.id), interval=15),
-            dict(attribute=str(self.benthic_attribute1a.id), interval=20),
-            dict(attribute=str(self.benthic_attribute1a.id), interval=25),
-            dict(attribute=str(self.benthic_attribute2c.id), interval=30),
+
+
+def test_fishbelt_protocol_validation_ok(
```

Comment:
Just for my own edification I'd be interested to know more about the move away from class-based tests.

Reply:
- fixtures
- lots of pytest plugins
- less boilerplate code